src/Map/src/Icon/MapIconRenderer.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Map\Icon;
+
+use Symfony\UX\Icons\IconRendererInterface;
+
+class MapIconRenderer implements MapIconRendererInterface
+{
+    public function __construct(
+        private readonly IconRendererInterface $renderer,
+    ) {
+    }
+
+    public function render(string $name): string
+    {
+        if (null === $this->renderer) {
+            throw new \LogicException('You must install the "symfony/ux-icons" package to use icons in the "ux_map" package. Run "composer require symfony/ux-icons" to install it.');
```

Comment:
Let's mimic Symfony's error message:
```suggestion
            throw new \LogicException('You cannot use an UX Icon as the "UX Icons" package is not installed. Try running "composer require symfony/ux-icons".');
```